### PR TITLE
Import and View Data Map Data

### PR DESCRIPTION
**Pull request by ghickman**
⚠️ Depends on #87 ⚠️ 

This adds importers for the missing CSV data and a view to replicate the `C. Data Map` sheet in the original Spreadsheet.

Some existing importers have been updated to take advantage of abstracted functionality (`strip_prefix`) and to account for small model variations.

A few common actions have been scripted to allow for easier dev testing.

The Data Map form looks like this by default:

and produces this output:

The number of rows in the table is static and only the tick/cross section is updated depending on the outcome of the filter.  The filter form uses `method="GET"` so all options are serialised to the URL for bookmarkability.

### CODE REVIEW

**Comment by ghickman** [comment]
Deployed to the QA server here: http://ec2-34-250-133-157.eu-west-1.compute.amazonaws.com/sudgt/data-map

File: phmi/management/commands/load_justification_details.py [hunk 1]
```diff
@@ -1,78 +1,94 @@
...
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--path", default="data/csvs/statutes", help="Path to load Statutes from"
         )
 
-        result = []
-        for i in legal_justification:
-            if prefix.lower() in i.name.split(":")[0].lower():
-                result.append(i)
+    # def get_justifications(self, justification_activity, org_type_name):
```
Comment (fredkingham) [inline phmi/management/commands/load_justification_details.py]: Do we really want to comment all the things? Can this just be deleted?

**Comment by fredkingham** [comment]
http://ec2-34-250-133-157.eu-west-1.compute.amazonaws.com/sudgt/activities seems broken. 

`NON-NHS PROVIDER (inc. charities) == Independent Sector`

I think I may have manually updated/renamed to the csvs to make this mapping...

File: phmi/management/commands/load_justification_details.py (hunk 1 above)
Comment (ghickman) [reply phmi/management/commands/load_justification_details.py]: Woops, didn't mean to leave those bits in, sorry about that!

**Comment by ghickman** [comment]
Latest changes deployed to http://ec2-34-250-133-157.eu-west-1.compute.amazonaws.com/sudgt/data-map

**Comment by fredkingham** [comment]
something is really not looking right

screen shot 2019-02-14 at 10 45 27
screen shot 2019-02-14 at 10 47 05

**Pull request merged**